process/interceptors/processor/trieNodeInterceptorProcessor.go

```diff
@@ -42,6 +42,11 @@ func (tnip *TrieNodeInterceptorProcessor) Save(data process.InterceptedData, _ c
 	return nil
 }
 
+// RegisterHandler registers a callback function to be notified of incoming trie nodes
+func (tnip *TrieNodeInterceptorProcessor) RegisterHandler(_ func(topic string, hash []byte, data interface{})) {
+	panic("not implemented")
```

Comment:
Instead of panic, we might print a log.Error as this is a non critical-programming error

Reply:
done